charts/scorpio-broker-dist/templates/entity-manager-deployment.yaml

```diff
@@ -0,0 +1,75 @@
+{{- if .Values.entityManager.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    service: {{ .Values.entityManager.entityLabel }}
+  name: {{ include "entity-manager.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+spec:
+  replicas: {{ .Values.entityManager.replicas }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "entity-manager.fullname" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  strategy:
+  {{- with .Values.entityManager.updateStrategy }}
+  {{- toYaml . | nindent 4 }}
+  {{- end }}
+  template:
+    metadata:
+      labels:
+        service: {{ include "entity-manager.fullname" . }}
+    spec:
+      containers:
+      - image: {{ .Values.entityManager.image.repository }}:{{ .Values.entityManager.image.tag }}
+        {{- if .Values.springArgs.overrideSpringArgs }}
+        env:
+          - name: spring_args
+            value: {{ .Values.springArgs.value }}
+        {{- end }}
+        imagePullPolicy: {{ .Values.entityManager.image.pullPolicy }}
+        name: {{ include "entity-manager.fullname" . }}
+        ports:
+        - containerPort: 1025
+          protocol: TCP
+        lifecycle:
+           preStop:
+            exec:
+              command:
+              - /bin/sh
+              - -c
+              - curl -X PUT {{ .Values.entityManager.eurekaAddress }}
+        {{- if .Values.entityManager.readinessProbe.enabled }}
+        readinessProbe:
+          exec:
+            command:
+            - sh
+            - -c
```

Comment:
Why dont you use an http check instead? It would allow you to get rid of curl inside the container.

Reply:
> @arpitNEC Hi, do you still work on this PR? There is still a number of open request and a couple of new once. Can you please check and resolve them? Let me know when I can review again. Best, Stefan

Hi @wistefan 
We have incorporated the review comments on this PR. Please review it again.
BR,
@RupamNEC